src/scale/Ordinal.ts

```diff
@@ -130,6 +130,10 @@ class OrdinalScale extends Scale<OrdinalScaleSetting> {
     }
 
     parse(val: OrdinalRawValue | OrdinalNumber): OrdinalNumber {
+        // Caution: Math.round(null) will return `0` rather than `NaN`
+        if (val == null) {
+            val = undefined;
```

Comment:
I think we can return `NaN` directly here

Reply:
Yeah, I didn't realize this.